deploy.php

```diff
@@ -0,0 +1,142 @@
+<?php
+namespace Deployer;
+
+require 'recipe/laravel.php';
+require 'recipe/rsync.php';
+require 'recipe/npm.php';
+
+// Project name
+set('application', 'meteo-raspi');
+
+// Configuration
+set('ssh_type', 'native');
+set('ssh_multiplexing', true);
+set('writable_mode', 'chmod');
+set('writable_chmod_mode', '0775');
+set('repository', 'https://github.com/matul91/meteo-raspi.git');
+set('keep_releases', 5);
+set('shared_files', [
+    '.env'
+]);
+set('shared_dirs', [
+    'storage',
+    'public/images/photos'
+]);
+set('writable_dirs', [
+    'bootstrap/cache',
+    'vendor',
+    'storage',
+    'public/images/photos'
+]);
+set('rsync', [
+    'exclude' => [
+        '.git',
+        'deploy.php',
+        'node_modules',
+    ],
+]);
+set('rsync_src', function () {
+    $local_src = get('local_release_path');
+    if (is_callable($local_src)) {
+        $local_src = $local_src();
+    }
+    return $local_src;
+});
+
+// Hosts
+
+host(getenv('DEPLOY_HOST'))
+    ->stage('production')
+    ->roles('app')
+    ->set('deploy_path', getenv('DEPLOY_PATH'))
+    ->set('branch', getenv('DEPLOY_BRANCH'))
+    ->user('deploy')
+    ->port(22)
+    ->identityFile('/tmp/deploy_rsa')
+    ->forwardAgent(true)
+    ->multiplexing(true)
+    ->addSshOption('UserKnownHostsFile', '/dev/null')
+    ->addSshOption('StrictHostKeyChecking', 'no');
+
+// Tasks
+task('npm:build', function () {
+    cd('{{release_path}}');
+    run('{{bin/npm}} run production');
+})->desc('Assets generation');
+
+task('production:migrate', function () {
+    run('{{bin/php}} {{release_path}}/artisan migrate:fresh --force');
+})->desc('Migrate clean database');
+
+task('production:seed', function () {
+    run('{{bin/php}} {{release_path}}/artisan db:seed --class=RolesTableSeeder --force');
+    run('{{bin/php}} {{release_path}}/artisan db:seed --class=ProductionUsersTableSeeder --force');
+    run('{{bin/php}} {{release_path}}/artisan db:seed --class=SettingsTableSeeder --force');
+})->desc('Seed database');
+
+task('passport:install', function () {
+    run('{{bin/php}} {{release_path}}/artisan passport:install --force');
+})->desc('Integrate passport');
+
+task('set:secret', function () {
+    run('{{bin/php}} {{release_path}}/artisan set:client_secret .env');
+})->desc('Set client secret in .env file');
+
+task('key:generate', function () {
+    run('{{bin/php}} {{release_path}}/artisan key:generate');
+})->desc('Generate application key');
+
+task('deploy', [
+    'deploy:prepare',
+    'deploy:lock',
+    'deploy:release',
+    'deploy:update_code',
+    'deploy:shared',
+    'deploy:vendors',
+    'deploy:writable',
+    'artisan:migrate',
+    'npm:install',
+    'npm:build',
+    'artisan:storage:link',
+    'artisan:view:clear',
+    'artisan:cache:clear',
+    'artisan:config:cache',
+    'artisan:optimize',
+    'deploy:symlink',
+    'deploy:unlock',
+    'cleanup',
+])->desc('Deploy project');
+
+task('initialize', [
```

Comment:
please rename to initializeEmptyProject, and remove redundant commands in a way that we can run normal deploy task and initilialize task consequently.

Reply:
Convention is to use single word name. For redundant command - there is specific order of them for each task (deploy and initialize), so it would have to be divided into several tasks (for start, then for deploy/initialize and then for finish) and it would cause more duplicated code (for checking if these tasks failed etc).